rapidpro/src/main/resources/securityRules.json

```diff
@@ -0,0 +1,19 @@
+[
+  {
+    "active": true,
+    "pattern": "/**/rapidpro/**",
+    "supportedSchemes": [
```

Comment:
but why should we keep no security? From my understanding this will lead to anyone being able to change my rapidpro configuration and obviously do nasty things  to me through this. 

If this is only for the settings UI, won't keeping the defaults be good enough? Or are there other controllers coming?